components/Template.tsx

```diff
@@ -0,0 +1,44 @@
+import Image from 'next/image';
+import pic1 from '../public/template/1.png';
+import pic2 from '../public/template/2.png';
+import pic3 from '../public/template/3.png';
+import pic4 from '../public/template/4.png';
+import pic5 from '../public/template/5.png';
+import pic6 from '../public/template/6.png';
+import { useInView } from 'react-intersection-observer';
+
+const Template = () => {
+  const { ref, inView } = useInView();
+
+  const arr = [
+    [pic1, 'animate-template1'],
+    [pic2, 'animate-template2'],
+    [pic3, 'animate-template3'],
+    [pic4, 'animate-template4'],
+    [pic5, 'animate-template5'],
+    [pic6, 'animate-template6'],
+  ];
+
+  return (
+    <div className="max-w-7xl mx-auto px-3 sm:px-6 lg:px-12 pt-12 pb-20 bg-template">
```

Comment:
should use single quote here I think

Reply:
It's still double quote even the prettier setting is single quote.